src/ophyd_async/epics/motor.py

```diff
@@ -42,7 +47,7 @@ class InvalidFlyMotorException(Exception):
 
 
 class FlyMotorInfo(BaseModel):
-    """Minimal set of information required to fly a motor:"""
+    """Minimal set of information required to fly a motor."""
```

Comment:
Are the `#:` comments here also used for docs? There are other places where they're explicitly converted to `"""`.

Reply:
No, `sphinx-autodoc2` doesn't support doc comments, so I needed to convert to `"""`. I may have missed some...